__init__.py

```diff
@@ -44,8 +45,6 @@
             print("The Retriever lacks permission to access the ~/.retriever/ directory.")
             raise
 SCRIPT_SEARCH_PATHS = [
```

Comment:
Didn't seem the need to have separate folders, since the new scripts get automatically moved to `~/.retriever/scripts` folder due to the CLI, and manually creating a JSON script is not recommended.